src/JoyOI.Monitor/Controllers/ManagementService/StateMachineController.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+using JoyOI.Monitor.Models;
+using System.Drawing;
+
+namespace JoyOI.Monitor.Controllers.ManagementService
+{
+    [Route("/ManagementService/StateMachine")]
+    public class StateMachineController : ChartController
+    {
+        const string MGMTSVC = "mgmtsvc";
+
+        [HttpGet("Created")]
+        public async Task<IActionResult> Created(int start, int end, int interval, int timezoneoffset)
+        {
+            if (start == 0 || end == 0 || interval == 0) {
+                Response.StatusCode = 400;
+                return Json(null);
+            }
+            var scaling = new ChartScaling(start, end, interval);
+            return Json(await GetChartData(
```

Comment:
Please make the line 26-34 in the following codes style:

```c#
@"SELECT
FLOOR (UNIX_TIMESTAMP(StartTime) / @interval) * @interval as t, 
Count(Id) as c 
......"
```

Reply:
Done